documentation/slate/source/includes/examples.md

```diff
@@ -100,7 +100,7 @@ In the ``links`` section there is for example this link: ``"href": "https://api.
 
 This link is about lines (according to its ``rel`` attribute) and is templated which means that it needs additional parameters.<br>
 The parameters are identified with the ``{`` ``}`` syntax.
-In this case it needs a region id. This id can the found in the ``regions`` section. 
+In this case it needs a region id. This id can the found in the ``regions`` section.
```

Comment:
I just remove the trailing whitespace.
I can fix it if you want but it's not the purpose of this PR :)